integration_test-process.txt

```diff
@@ -237,3 +237,14 @@ Extended testing
   + instance has repos
   + cat /etc/hosts server point to an update server in the region
   + zypper up has no errors
+
+Orphaned credentials
+On SLE 15 PAYG
+- registercloudguest
+  + check all repos are there
+  + configure the instance/env so it can't access or verify on any sibling on the update infrastructure
+    + possible best approach is:
+      pointing the instance to test infra and edit test infra for this test
+  + rm /etc/products.d/$SOME_PRODUCT
```

Comment:
This would not orphan credentials. The `__get_referenced_credentials()` function is driven based on repository setup, not based on installed products. A `.repo` will have a refernce to a credentais file in the URL, hence the name of the function. So the test should be to remove a `.repo` file for a specific product. No restriction to access the infra is needed as zypper is not executed in this path. An access restricted environment with `--clean` is yet another test case and for that why not use the `ip` command to simply drop packages, there's an example for this in the doc already in the fail over testing.